lib/defra_ruby/validators/companies_house_number_validator.rb

```diff
@@ -25,14 +25,14 @@ def validate_each(record, attribute, value)
       def value_is_present?(record, attribute, value)
         return true if value.present?
 
-        record.errors[attribute] << error_message(:company_no, :blank)
+        record.errors[attribute] << error_message(:attribute, :blank)
```

Comment:
If this is always going to be `:attribute`, shall we change the `#errror_message` implementation to only accept the second param and hard-code `:attribute` there?

Reply:
Good shout! I'll fix it.

Reply:
@cintamani Ended up abandoning `attribute` entirely: https://github.com/DEFRA/defra-ruby-validators/pull/42/commits/48cbe834f86699034474f7211c8521673228a8a6

Reply:
Even better! :D